chain/src/main/scala/org/bitcoins/chain/models/CompactFilterDAO.scala

```diff
@@ -119,10 +119,48 @@ case class CompactFilterDAO()(implicit
   }
 
   /** Gets the heaviest filter from the database */
+  def getBestFilter1: Future[Option[CompactFilterDb]] = {
+    for {
+      blockHeaderDbs <- BlockHeaderDAO().chainTips
+      filterHeaderOpt <-
+        if (blockHeaderDbs.isEmpty) {
+          Future.successful(None)
+        } else {
+          val blockHeaderDb = blockHeaderDbs.maxBy(_.chainWork)
+          findByBlockHash(blockHeaderDb.hashBE)
+        }
+    } yield filterHeaderOpt
+  }
+
+  implicit private val bigIntMapper: BaseColumnType[BigInt] =
```

Comment:
This should probably live some where in `dbCommons` no? Is this specific to `CompactFilterDAO`?

Reply:
These are needed for the `BlockHeaderDAO`, for the chain work we have different ways to serialize the `BitInt` for the db for postgres vs sqlite. We need to set the mapper based off of the config, but we can't get access to the config in `dbCommons` so we need to set it here. These were there beforehand I just had moved them on accident.